bin/dockerfilelint

```diff
@@ -4,21 +4,25 @@ var process = require('process');
 var fs = require('fs');
 var os = require("os");
 var path = require('path');
-var argv = require('yargs').argv;
+var argv = require('yargs')
+            .alias('o', 'output')
+            .describe('output', 'the format in which you want to display the lint results (default is cli)')
```

Comment:
Nice and clean. I didn't dive _too_ deep into what I could do with `yargs` so I left it simple. Happy to use the above!

I'll be sure to update the README as well